custom_components/yamaha_ynca/media_player.py

```
@@ -641,21 +637,20 @@ def build_presetlist_media_item(
             name = "TUNER (DAB)"
         elif media_content_id_type == "fmpresets":
             name = "TUNER (FM)"
-        else:
-            if subunit := getattr(self._ynca, subunit_attribute_name, None):
-                input = InputHelper.get_input_for_subunit(subunit)
-                source_mapping = InputHelper.get_source_mapping(self._ynca)
-                name = source_mapping.get(input, source_mapping[input])
+        elif subunit := getattr(self._ynca, subunit_attribute_name, None):
+            input = InputHelper.get_input_for_subunit(subunit)
```

Ruff (0.8.2)

Variable `input` is shadowing a Python builtin

(A001)